main.tf

```diff
@@ -24,17 +24,26 @@ resource "local_file" "cloud-datastore-index-file" {
   filename = local.path_file
 }
 
+resource "google_app_engine_application" "app" {
```

Comment:
What is the motivation for adding this resource?

Reply:
Okay, let's move this resource to the example and document that it is required in the README. This module should not be responsible for configuring App Engine.